src/chronometer.js

```diff
@@ -1,33 +1,60 @@
 class Chronometer {
   constructor() {
+  this.currentTime = 0;
+  this.intervalId = null;
+  
     // ... your code goes here
   }
 
   start(printTimeCallback) {
-    // ... your code goes here
+     this.intervalId = setInterval(() => {
+      this.currentTime++;
+      if (printTimeCallback) {
+        printTimeCallback(); 
+      }}, 1000);
+    
   }
+    // ... your code goes here
+  
+  
 
   getMinutes() {
+    let numberofMinutes = Math.floor(this.currentTime/60);
+    return numberofMinutes;
     // ... your code goes here
   }
 
   getSeconds() {
+
+    let numberofSeconds = this.currentTime % 60;
+    return numberofSeconds;
+   
     // ... your code goes here
   }
 
   computeTwoDigitNumber(value) {
```

Comment:
Here you shouldn't redefine the `value` and you can put the methods together and after return the final result

```
 computeTwoDigitNumber(value) {

    // value = this.currentTime    
    const string = value.toString().padStart(2, '0');
    
      return string;
    // ... your code goes here

  }
```